xml/templates/civicrm_state_province.tpl

```diff
@@ -4175,7 +4172,15 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (NULL, 1126, 'RD', 'Redange-sur-Attert'),
 (NULL, 1126, 'RM', 'Remich'),
 (NULL, 1126, 'VD', 'Vianden'),
-(NULL, 1126, 'WI', 'Wiltz')
+(NULL, 1126, 'WI', 'Wiltz'),
+
+-- Add missing provinces for Zambia
```

Comment:
I'm not sure what information this comment conveys. Obviously every single `git commit` adds something to CiviCRM that was previously missing, but as a general rule we communicate about it in the commit messages rather than cluttering up the code itself with comments about what was committed.

Reply:
It would be nice to "rebase" this file, and removing the hardcoded IDs everywhere, and fold back the "missing" provinces into the main set of provinces. For now I was just following [tradition](https://www.youtube.com/watch?v=gRdfX7ut8gw) :notes: 
I can do cleanup, but would rather do it in a separate PR, because the upgrade bit will quickly need a rebase. The cleanup will take time.

Reply:
Ok I'll follow tradition and merge this